settings/src/components/revalidate-modal.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * WordPress dependencies
+ */
+import { useContext, useEffect, useRef, useState } from '@wordpress/element';
+import { GlobalContext } from '../script';
+import { Button, Modal, __experimentalHStack as HStack } from '@wordpress/components';
```

Comment:
I saw talk of this being added to stable as it's used `everywhere` now. Can probably eslint ignore this particular failure.

Reply:
That's what I was thinking, it appears to be one of the "stable" unstable API's now, and why I used it in the first place.

I'm not sure how to mark it as ignored though yet.

Reply:
Changed my mind as it looks like [things are still unstable](https://github.com/WordPress/gutenberg/issues/43085)